docker/local-metadata/docker-compose.yml

```diff
@@ -59,6 +59,7 @@ services:
     container_name: openmetadata_server
     environment:
       # Elasticsearch configuration
+      OPENMETADATA_DEBUG: ${OPENMETADATA_DEBUG:-false}
```

Comment:
Can we add this for all the docker-compose.yml files ?
development -
- docker/local-metadata/docker-compose-postgres.yml

production -
- docker/metadata/docker-compose.yml
- docker/metadata/docker-compose-postgres.yml

Reply:
No only for dev